ide/go.lang/nbproject/project.properties

```diff
@@ -16,7 +16,10 @@
 # under the License.
 javac.source=1.8
 
-# As of 2nd Jan 2023, we use the Grammar from:
-# https://github.com/antlr/grammars-v4/tree/4794c0c5371ce6b0711a306256f8f7706394ec6d/golang
+# As of 14th April 2024, we use the Grammar from:
+# https://github.com/antlr/grammars-v4/tree/bcc9ec1b35e4ede2f90abdb8713a56315a634a92/golang
 #
-golang.grammar.version=4794c0c5371ce6b0711a306256f8f7706394ec6d
+golang.grammar.version=bcc9ec1b35e4ede2f90abdb8713a56315a634a92
+
+# Simple language update can result too many signature changes in the generated code.
+sigtest.skip.gen=true
```

Comment:
Why drop the pretense, that this is module has an API and require that users make dependencies implementation dependencies?

Reply:
Agreed!  Although I'm also curious why this is a module at all?